src/admin-views/aggregator/origins/refine.php

```diff
@@ -57,24 +44,25 @@
  *
  * @param array $keyword_exclusions List of origins excluded.
  */
-$keyword_exclusions = json_encode( apply_filters( 'tribe_events_aggregator_refine_keyword_exclusions', $keyword_exclusions ) );
+$keyword_exclusions = apply_filters( 'tribe_events_aggregator_refine_keyword_exclusions', [ 'facebook' ] );
+$keyword_exclusions = json_encode( $keyword_exclusions );
+
 /**
  * Allow filtering of origins excluded from refining EA results by location.
  *
  * @since 4.6.24
  *
  * @param array $location_exclusions List of origins excluded.
  */
-$location_exclusions = json_encode( apply_filters( 'tribe_events_aggregator_refine_location_exclusions', $location_exclusions ) );
-
-
+$location_exclusions = apply_filters( 'tribe_events_aggregator_refine_location_exclusions', [ 'url', 'facebook' ] );
+$location_exclusions = json_encode( $location_exclusions );
```

Comment:
:warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: 97184ad1e6b46cf48bf27abb91f0f458</sub>